support/locust/install.py

```diff
@@ -0,0 +1,3 @@
+import sys
```

Comment:
What's this for? Do we need this file?

Reply:
removed, was added for small fix but resolved later